auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -70,9 +71,8 @@ public FacebookProvider(Context appContext, IdpConfig idpConfig) {
         } else {
             mScopes = scopes;
         }
-        String applicationId = appContext.getString(R.string.facebook_application_id);
         FacebookSdk.sdkInitialize(appContext);
-        FacebookSdk.setApplicationId(applicationId);
+        FacebookSdk.setWebDialogTheme(theme);
```

Comment:
This fixes the issue where the theme wasn't applied to the webview for some reason:

# Old
![screenshot_20161205-165327](https://cloud.githubusercontent.com/assets/9490724/20909986/aa77933e-bb13-11e6-8568-62df3948cace.png)

# New
![screenshot_20161205-164658](https://cloud.githubusercontent.com/assets/9490724/20909991/b36c28ec-bb13-11e6-9bc6-c10c3085a653.png)